2024/json-ld-wg/index.html

```diff
@@ -153,7 +153,11 @@ <h2>Motivation and Background</h2>
       <section id="scope" class="scope">
         <h2>Scope</h2>
 				<p>
-					The Working Group will maintain the JSON-LD specifications (i.e., <a href="https://www.w3.org/TR/json-ld11/">JSON-LD 1.1</a>, <a href="https://www.w3.org/TR/json-ld11-api/">JSON-LD 1.1 API</a>, <a href="https://www.w3.org/TR/json-ld11-framing/">JSON-LD 1.1 Framing</a>) that together provide a JSON format for Linked Open Data to interoperate at web-scale, in a method which is familiar to and usable by web-focused software engineers.
+					The Working Group will extend the recommendations defining the JSON-LD specifications (i.e., <a href="https://www.w3.org/TR/json-ld11/">JSON-LD 1.1</a>, <a href="https://www.w3.org/TR/json-ld11-api/">JSON-LD 1.1 API</a>, <a href="https://www.w3.org/TR/json-ld11-framing/">JSON-LD 1.1 Framing</a>)
+          with the features introduced by <a href="https://www.w3.org/TR/rdf12-concepts">RDF-star</a>
+          in addition to open Errata and requested features
+          as noted on the groups <a href="https://github.com/orgs/w3c/projects/84">Project Management Page</a>.
```

Comment:
This may need to be specific and/or refer to a static page, rather than a dynamically updated page.